--- ipython-0.11/IPython/zmq/__init__.py.orig	2011-07-21 23:21:07.000000000 +0200
+++ ipython-0.11/IPython/zmq/__init__.py	2011-10-25 05:45:04.213115630 +0200
@@ -12,6 +12,7 @@
 import warnings
 
 minimum_pyzmq_version = "2.1.4"
+minimum_pyzmq = (2, 1, 4)
 
 try:
     import zmq
@@ -20,7 +21,7 @@
 
 pyzmq_version = zmq.__version__
 
-if pyzmq_version < minimum_pyzmq_version:
+if zmq.zmq_version_info() < minimum_pyzmq:
     raise ImportError("IPython.zmq requires pyzmq >= %s, but you have %s"%(
                     minimum_pyzmq_version, pyzmq_version))
 
