diff -Nur ipython-0.13.orig/IPython/core/release.py ipython-0.13/IPython/core/release.py
--- ipython-0.13.orig/IPython/core/release.py	2012-06-30 03:40:24.000000000 +0100
+++ ipython-0.13/IPython/core/release.py	2012-07-24 01:21:16.000000000 +0100
@@ -114,7 +114,7 @@
            'Brian'    : ('Brian E Granger', 'ellisonbg@gmail.com'),
            'Min'      : ('Min Ragan-Kelley', 'benjaminrk@gmail.com'),
            'Thomas'   : ('Thomas A. Kluyver', 'takowl@gmail.com'),
-           'Jörgen'   : ('Jörgen Stenarson', 'jorgen.stenarson@bostream.nu'),
+           'Jorgen'   : ('Jorgen Stenarson', 'jorgen.stenarson@bostream.nu'),
            'Matthias' : ('Matthias Bussonnier', 'bussonniermatthias@gmail.com'),
            }
 
